domains/logistics/configure-organization/README.md

```diff
@@ -2,56 +2,60 @@
 
 ## Description
 
-The FL Logistics has a depot in Lyon and another in Saint-Étienne. Each depot has one _receiving_ and one _picking_ setting. Each _receiving_ setting has one forklift, one pallet jack, and one barcode reader. Each _picking_ setting has one forklift, one pallet jack, and one label printer. Only one employee can be associated with the usage of one setting, but the same employee can be associated with the usage of multiple settings.
+The FL Logistics has a depot in Lyon and another in Saint-Étienne. Each depot has one _receive_ and one _pick_ setting.
 
-Forklifts are capable of lifting up and lifting down materials on pallets to/from high shelves, as well as moving them from one location to another. Pallet jacks are capable of lifting up and lifting down materials on pallets placed on the floor and move them from one location to another. They both are used for loading and unloading trucks by moving materials in the _receiving_ and the _picking_ settings.
+In the _receive_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets, and list the products in a pallet based on barcode.
```

Comment:
> based on barcode

based on their barcode?

Reply:
Replaced by

In the _receive_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets and can read barcodes identifying the pallets' content.